src/modules/configurator/components/ConfigOptionsDisplay.tsx

```diff
@@ -0,0 +1,48 @@
+import { deleteUserConfiguration } from "../services";
+import { useNavigate, useSearchParams } from "react-router-dom";
+import { useRecoilValue } from "recoil";
+import { userState } from "../../authentification/state";
+import { notifyDelete } from "../utilities/utilities";
```

Comment:
Check this folder structure

Reply:
Utilities folder and file are of the same name, changed file name to utils to avoid confusion.